root/edit/details/edit_medium.tt

```diff
@@ -199,13 +199,11 @@
                  </td>
                  <td><span class="diff-only-a">[% IF old_track.recording; descriptive_link(old_track.recording); END %]</span></td>
                  <td>
-                   <span class="diff-only-b">
-                     [% # If no recording_id exists, it's creating a recording
-                       SET allow_new = 1 IF !new_track.recording_id
-                     %]
-                     [% descriptive_link(new_track.recording) %]
-                     [% SET allow_new = 0 IF !new_track.recording_id %]
-                   </span>
+                   [% # If no recording_id exists, it's creating a recording
+                     SET allow_new = 1 IF !new_track.recording_id
+                   %]
+                   <span class="diff-only-b">[% descriptive_link(new_track.recording) %]</span>
+                   [% SET allow_new = 0 IF !new_track.recording_id %]
```

Comment:
To remove whitespaces and line breaks inside the `td` tag:

```suggestion
                   [%~ # If no recording_id exists, it's creating a recording
                     SET allow_new = 1 IF !new_track.recording_id
                   ~%]
                   <span class="diff-only-b">[% descriptive_link(new_track.recording) %]</span>
                   [%~ SET allow_new = 0 IF !new_track.recording_id ~%]
```

See [chomp tag modifiers in Template Toolkit](http://template-toolkit.org/docs/manual/Config.html#section_PRE_CHOMP_POST_CHOMP).

That would work with the original code too.